### ISSUE TITLE
Consider rebranding to “Learn Astro Code”

### ISSUE THREAD

**Issue by kelle** [astropy/astropy-tutorials#413]
Learn Astro Code

Learn Astropy implies we don’t accept materials relevant to astro but not using astropy.

Tagline - resources for teaching and learning code relevant to Astronomy.

**Comment by kakirastern** [comment]
Yeah, we could have the "Learn Astro Code" brand under "Learn Astropy"... Maybe best to best keep site name as `learn.astropy.org` for "Learn Astropy" to show affiliation with `Astropy`, but with the brand "Learn Astro Code" for general astronomy education purposes regarding coding specifically.
